crypto_secretstream/Cargo.toml

```
@@ -0,0 +1,28 @@
+[package]
+name = "crypto_secretstream"
+version = "0.1.0"
+description = """
+Pure Rust implementation of libsodium's crypto_secretstream secret-key using
+chacha20 and poly1305
```

When referring to the algorithms, I think it makes sense to capitalize them:

```suggestion
ChaCha20 and Poly1305
```